packages/cardpay-sdk/contracts/addresses.ts

```diff
@@ -89,6 +95,9 @@ const XDAI = {
   uniswapV2Router: '0x1C232F01118CB8B424793ae03F870aa7D0ac7f77', // This is the UniswapV2Router02
   uniswapV2Factory: '0xA818b4F111Ccac7AA31D0BCc0806d64F2E0737D7',
   rewardPool: '0x42628325845B3e01EfdD6ce4b3665453dC6c9A13',
+  // rewardManager: '',
+  // registerRewardProgramHandler: '',
+  // registerRewardeeHandler: '',
```

Comment:
don't comment these out--just keep them in there with empty values. the error messages that we'll receive are useful